state-chain/chains/Cargo.toml

```diff
@@ -54,12 +55,18 @@ anyhow = { workspace = true, optional = true }
 hex = { workspace = true, features = ["serde"] }
 hex-literal = { workspace = true }
 serde = { workspace = true, features = ["derive", "alloc"] }
+serde_bytes = { workspace = true }
+serde-big-array = { workspace = true }
 log = { workspace = true }
 itertools = { workspace = true }
 arrayref = { workspace = true }
 strum = { workspace = true }
 thiserror = { workspace = true, optional = true }
 rand = { workspace = true, optional = true }
+derive-where = "1.2.7"
+proptest = { version = "1.6", optional = true } # note this has to be in the normal dependencies because other packages depend on it
+duplicate = "2.0.0"
+saturating_cast = "0.1.0"
```

Comment:
Are these no-std by default? So we need an entry in the std feature section below?